### DISCUSSION TITLE
Which Redis versions does Lettuce 5.x support?

### DISCUSSION THREAD

**Discussion by yunbozhang-msft** [redis/lettuce#2367] [Q&A] [answered]
Hi team, is there any introduction to the Redis version supported by Lettuce 5.x?
I see that Lettuce 6 supports Redis 2.6+ up to Redis 7.x, but there isn't any compatibility documentation for Lettuce 5. Thanks！

![image](https://user-images.githubusercontent.com/44389598/227435497-5ac73521-9abc-493c-8ece-ecb2cee55d59.png)

**Comment by mp911de** [comment, marked as answer]
Duplicate of #2350
